packages/ocamlfind/ocamlfind.1.9.0/opam

```diff
@@ -37,14 +36,19 @@ install: [
   [make "install"]
   ["install" "-m" "0755" "ocaml-stub" "%{bin}%/ocaml"] {ocaml:preinstalled}
 ]
-extra-files: [
-  ["fix-bsd.patch" "md5=00e4db61fceee51af794c3a36e4cef0e"]
-]
 dev-repo: "git+https://gitlab.camlcity.org/gerd/lib-findlib.git"
+extra-source "ocaml-stub" {
+  src: "https://raw.githubusercontent.com/ocaml/ocamlfind/findlib-1.9/ocaml-stub"
+  checksum: [
+    "md5=181f259c9e0bad9ef523e7d4abfdf87a"
+    "sha512=d838b025311fe3f3dc887884a741749eebd2878a7b04562a216c8a6a7b435a35bd178cd8b4dab8abf117e4709a722368ec2960198423da6018a8bdb30f590b7a"
+  ]
+}
```

Comment:
The `ocamlfind.install` file is also to be downloaded.

Reply:
Good point!

Reply:
That of course didn't fail on my FreeBSD 11 VM!